hazelcast/src/test/java/com/hazelcast/dataconnection/impl/ListResourcesTest.java

```diff
@@ -72,6 +73,13 @@ public static Object[] params() {
                                 new DataConnectionResource("TABLE", "my_schema", "my_table")
                         }
                 },
+                {
+                        new MSSQLDatabaseProvider(),
+                        new DataConnectionResource[]{
+                                new DataConnectionResource("TABLE", "master", "dbo", "my_table"),
+                                new DataConnectionResource("TABLE", "master", "my_schema", "my_table")
```

Comment:
Let's use `OBJECT_TYPE_TABLE` as the first parameter.

Reply:
rev : ad88a2e283ee498ff655caaac0dc0658e55bab5e